TTHAnalysis/python/plotter/COEF/card/costhetaW.card.txt

```diff
@@ -0,0 +1,40 @@
+## Datacard for cut file RunII_SM_WZ/2016/cuts_wzsm.txt (signal prompt_WZ_other)
```

Comment:
Please, don't commit datacards to the repository (in any case just the command to run them). Remove it from the remote by using:
git rm --cached TTHAnalysis/python/plotter/COEF/card/costhetaW.card.txt

Reply:
Same for the .root file